File: sdk/csharp/src/AnyVali/Interchange/Exporter.cs
namespace AnyVali.Interchange;

/// <summary>
/// Export a schema to an AnyValiDocument.
/// </summary>
public static class Exporter
{
    public static AnyValiDocument ExportSchema(Schema schema, ExportMode mode = ExportMode.Portable)
    {
        return schema.Export(mode);
    }
}

File: sdk/js/src/interchange/exporter.ts
import type { AnyValiDocument, ExportMode } from "../types.js";
import { BaseSchema } from "../schemas/base.js";

/**
 * Export a schema to a portable AnyValiDocument.
 */
export function exportSchema(
  schema: BaseSchema,
  mode: ExportMode = "portable"
): AnyValiDocument {
  return schema.export(mode);
}
